packages/dev/src/LearningPathCatalog.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import {
+  QuickStart,
+  QuickStartCatalog,
+  QuickStartCatalogEmptyState,
+  QuickStartCatalogFilterCountWrapper,
+  QuickStartCatalogFilterSearchWrapper,
+  QuickStartCatalogFilterStatusWrapper,
+  QuickStartCatalogHeader,
+  QuickStartCatalogSection,
+  QuickStartCatalogToolbar,
+  QuickStartContext,
+  QuickStartContextValues,
+  QuickStartTile,
+  clearFilterParams,
+  filterQuickStarts,
+  getQuickStartStatus,
+} from '@patternfly/quickstarts';
+import {
+  Divider,
+  Gallery,
+  GalleryItem,
+  Text,
+  TextContent,
+  ToolbarContent,
+} from '@patternfly/react-core';
+
+export const LearningPathCatalog: React.FC = () => {
```

Comment:
looks like mostly the same as the custom catalog? can we reuse it and pass in the filter criteria?